core/Test/performance_test.cpp

```diff
@@ -340,7 +340,7 @@
 //     end = std::chrono::high_resolution_clock::now();
 //     elapsed = std::chrono::duration_cast<std::chrono::nanoseconds>(end - begin);
 
-//     printf("Time measured of Sign(RSA_4096) with Repeat NUM(%d): %.6f seconds.\n", PERF_NUM, elapsed.count() * 1e-9);
```

Comment:
commit title
"
Add prefix "EH_" for each keyspec
"

Reply:
fixed